bundles/org.openhab.binding.automower/README.md

```diff
@@ -139,44 +137,40 @@ The following actions are available for `automower`things:
 | park                   | duration (int) | Parks the automower for the given duration (minutes), overriding the schedule                  |
 | resumeSchedule         | -              | Resumes the schedule for the automower                                                         |
 
-
 ## Full Example
 
 ### automower.thing
 
-	Bridge automower:bridge:mybridge [ appKey="<your_private_application_key>", userName="<your_username>", password="<your_password>" ] {
-			Thing automower myAutomower [ mowerId="<your_id_received_from_discovery>", pollingInterval=3600] {
-		}
-	}
-
+ Bridge automower:bridge:mybridge [ appKey="<your_private_application_key>", userName="<your_username>", password="<your_password>" ] {
```

Comment:
Looks like a ```java code block  is missing here too.

Reply:
Maybe the files are throwing too much errors. 🤷‍♂️
I have checked it locally and it gets displayed as an error.

No idea why it was missed so far.